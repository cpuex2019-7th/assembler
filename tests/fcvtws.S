    li t1, 1068708659           ; 1.4
    fmvwx ft1, t1               

    fcvtws t1, ft1              ; t1 should be 1

    li t2, 1080452710           ; 3.6
    fmvwx ft2, t2
    
    fcvtws t2, ft2              ; t1 should be 4

    add a0, t1, t2             ; a0 = t1 + t2, a0 should be 5
