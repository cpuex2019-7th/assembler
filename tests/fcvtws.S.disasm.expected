lui t1, 260915
addi t1, t1, 819
fmvwx ft1, t1
fcvtws t1, ft1
lui t2, 263782
addi t2, t2, 1638
fmvwx ft2, t2
fcvtws t2, ft2
add a0, t1, t2
